tests/cases/lib/cps/cps-path-not-set-root.cps

```diff
@@ -0,0 +1,13 @@
+{
+    "Name": "cps-path-not-set-root",
```

Comment:
Probably another PR, but didn't we switch to lower case field names?

Reply:
We have #11 open for that. I'd prefer to do one big swoop and convert everything than do it piece meal